src/connectors/withHistoryActions.jsx

```diff
@@ -17,12 +17,18 @@ class WithHistoryActions extends Component {
     otherProps: {},
   };
 
-  handlePush = (pathname) => {
-    this.props.historyPush({ pathname });
+  handlePush = (pathname, params = { state: {}, silent: false }) => {
+    this.props.historyPush({
+      pathname,
+      params,
```

Comment:
While reading how historyPush/replace is designed I'd say it will not work. Did you try this code if it works?

Reply:
Updated this function after testing. Silent option does not work because of a bug in the pwa. Spoke with Flo about the bug I encountered with the silent param and he advised me to not use it.